src/main/scala/xiangshan/cache/mmu/PageTableWalker.scala

```diff
@@ -535,15 +644,27 @@ class LLPTWIO(implicit p: Parameters) extends MMUIOBaseBundle with HasPtwConst {
       val h_resp = Output(new HptwResp)
     }))
   }
+  val bitmap = new Bundle{
+    val req = DecoupledIO(new bitmapReqBundle())
+    val resp = Flipped(DecoupledIO(new bitmapRespBundle()))
+  }
+
```

Comment:
Should be a option bundle? Then we need not say:
```
  if (HasBitmapCheck) {
    io.bitmap.xxx := xxx
  } else {
    io.bitmap.xxx := DontCare
  }
```

If we don't set these bundles as options, we can only rely on chisel's functionality to optimise them away; not that they won't be generated initially